tests/blockchain/eth/contracts/base/test_dispatcher.py

```diff
@@ -393,6 +397,78 @@ def test_dispatcher(testerchain):
     assert 1 == len(events)
     assert 33 == events[0]['args']['value']
 
+    # Check upgrading to the contract with explicit storage slots
```

Comment:
Very cool @szotov .  Is there a slot reference that we can link in a comment?

Reply:
https://solidity.readthedocs.io/en/latest/miscellaneous.html#layout-of-state-variables-in-storage